packages/jest-jasmine2/src/reporter.js

```diff
@@ -4,39 +4,71 @@
  * This source code is licensed under the BSD-style license found in the
  * LICENSE file in the root directory of this source tree. An additional grant
  * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @flow
  */
 'use strict';
 
+import type {Config} from 'types/Config';
```

Comment:
I had the realization that this file only needs the config so it can pass it down to `JasmineFormatter`. I was expecting it to need the `{noHighlight, noStackTrace}` options, but that wasn't the case.

We still could lock the config down to just the option `JasmineFormatter` needs. Thoughts?